pylint jogar_novamente_screen.py

************* Module jogar_novamente_screen
src/jogar_novamente_screen.py:17:0: C0303: Trailing whitespace (trailing-whitespace)
src/jogar_novamente_screen.py:1:0: C0114: Missing module docstring (missing-module-docstring)
src/jogar_novamente_screen.py:7:0: C0116: Missing function or method docstring (missing-function-docstring)
src/jogar_novamente_screen.py:29:29: E1101: Module 'pygame' has no 'QUIT' member (no-member)

--- a/src/jogar_novamente_screen.py
+++ b/src/jogar_novamente_screen.py
@@ -1,10 +1,28 @@
+"""
+Module that defines the screen displayed when the player chooses to play again.
+
+This module handles the logic and rendering of the 'play again' screen,
+including displaying the background, button, and handling user interactions.
+"""
+
 from os import path
 import pygame
 from assets import VITORIA_BG, load_assets, BOTAO_JOGARN2
 from sprites import Button
 from config import HEIGHT,WIDTH,IMG_DIR, FPS, BLACK, QUIT, GAME, SND_DIR
 
+
 def jogar_novamente_screen(window):
+    """
+    Display the 'Play Again' screen where the user can restart the game or quit.
+
+    Args:
+        window: The Pygame window where the screen will be rendered.
+
+    Returns:
+        state: The state of the game after the user interaction (QUIT or GAME).
+    """
+
     clock = pygame.time.Clock()
 
     assets = load_assets()
@@ -13,8 +31,8 @@ def jogar_novamente_screen(window):
     background_rect = background.get_rect()
 
     img_botao = assets[BOTAO_JOGARN2]
-    botao = Button(WIDTH/2-225, HEIGHT/2+0, img_botao, 0.5)
-    
+    botao = Button(WIDTH / 2 - 225, HEIGHT / 2 + 0, img_botao, 0.5)
+
     swoosh = pygame.mixer.Sound(path.join(SND_DIR, 'swoosh_de_terror.wav'))
     swoosh.play()
     running = True
@@ -25,7 +43,6 @@ def jogar_novamente_screen(window):
         botao.draw(window)
 
         for event in pygame.event.get():
-            # Verifica se foi fechado.
             if event.type == pygame.QUIT:
                 state = QUIT
                 running = False

